java/clients/src/main/java/sleeper/clients/admin/AdminClientStatusStoreFactory.java

```diff
@@ -58,6 +63,11 @@ public IngestJobStatusStore loadIngestJobStatusStore(InstanceProperties instance
             public IngestTaskStatusStore loadIngestTaskStatusStore(InstanceProperties instanceProperties) {
                 return IngestTaskStatusStoreFactory.getStatusStore(dynamoDB, instanceProperties);
             }
+
+            @Override
+            public IngestBatcherStore loadIngestBatcherStatusStore(InstanceProperties properties, TablePropertiesProvider tablePropertiesProvider) {
+                return new DynamoDBIngestBatcherStore(dynamoDB, properties, tablePropertiesProvider);
```

Comment:
Maybe it would be worth returning an optional here? We could check whether the optional stack is enabled, and return an empty optional if it isn't. Please add a test for this case.